From a5bb9fcc8cb255c015e5fe6d5e14afb9f0f26693 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan-Simon=20M=C3=B6ller?= <dl9pf@gmx.de>
Date: Tue, 20 Nov 2012 21:21:12 +0100
Subject: [PATCH 1003/1005] disable-clang-integrated-as.patch

Disable the use of the Clang Integrated Assembler

This patch probably can't be sent upstream in this form
This patch needs to be reworked

Signed-off-by: Mark Charlebois <charlebm@gmail.com>

// Completely disable the integrated-as. //

---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- linux.orig/Makefile
+++ linux/Makefile
@@ -383,11 +383,11 @@
 KBUILD_CFLAGS   := -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs \
 		   -fno-strict-aliasing -fno-common \
 		   -Werror-implicit-function-declaration \
-		   -Wno-format-security \
+		   -Wno-format-security $(call cc-option,-no-integrated-as,)\
 		   -fno-delete-null-pointer-checks
 KBUILD_AFLAGS_KERNEL :=
 KBUILD_CFLAGS_KERNEL :=
-KBUILD_AFLAGS   := -D__ASSEMBLY__
+KBUILD_AFLAGS   := -D__ASSEMBLY__ $(call cc-option,-no-integrated-as,)
 KBUILD_AFLAGS_MODULE  := -DMODULE
 KBUILD_CFLAGS_MODULE  := -DMODULE
 KBUILD_LDFLAGS_MODULE := -T $(srctree)/scripts/module-common.lds
